src/main.ts

```diff
@@ -6,7 +6,7 @@ import store from './store/index.js'
 import { i18n } from './i18n'
 import VueFormatters from './lib/formatters'
 import packageJSON from '../package.json'
-import { vuetify } from '@/plugins/vuetify'
+import { vuetify } from '@/plugins/vuetify.ts'
```

Comment:
why you added import extension?

Reply:
It seems `[fix: vite cannot resolve typescript files](https://github.com/Adamant-im/adamant-im/pull/499/commits/ca78437c76ef017b1f1a8afca4d639860232c0de)`

Reply:
Solved by resolving `.ts` extension in vite.config.ts https://github.com/Adamant-im/adamant-im/pull/499/commits/2fbe4663b064443c205cf5b49ed876b1abde76c6